### DISCUSSION TITLE
Welcome to gorm Discussions!

### DISCUSSION THREAD

**Discussion by jinzhu** [go-gorm/gorm#5523] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tomburgs** [comment]
Hei 👋 
How nice that we now have GitHub discussions!

I'll start — I'm Toms, enjoying my time here in the sunny city of Copenhagen ☀️
I've been using gorm for a while in different projects and it's one of my favorite ORM libraries (not exclusive to Go) and I keep coming back to it.

Of course I also picked gorm for my latest endeavor and it has proven to be a solid choice, allowing me to quickly develop my project (won't plug it here, but one can easily discover it by visiting my profile). My favorite feature must be the auto migrations as they are great for rapidly prototyping.

Excited to connect with others and share/hear some gorm tips!

**Comment by MmToon** [comment]
gorm:"column:a.b"
I want to specify the table name when specifying the field, which can save a lot of repetitive code. Reduce the use of select
